### DISCUSSION TITLE
Request for NUGET_ORG_TOKEN_2020 secret

### DISCUSSION THREAD

**Discussion by jcoliz** [fsprojects/FsProjectsAdmin#39] [General]
Hello, fsprojects-admins. I would like to release an updated package for ExcelFinancialFunctions to the NuGet Gallery. Doing so will require access to the nuget gallery publishing secret. Could someone with the necessary access please establish this secret in the ExcelFinancialFunctions repo?

Thanks so much!

**Comment by dsyme** [comment]
Yup, I'll get it sorted

**Reply by jcoliz** [reply]
Thanks so much!

**Comment by dsyme** [comment]
I've done it now, though the secret is called `NUGET_ORG_TOKEN` (there was no sense to use the 2020 suffix originally )

**Comment by dsyme** [comment]
@sergey-tihon Note that each repo gets its own publishing token, limited to only the packages/glob for that repo. We should check you have the rights to create them. 

They last one year and so we'll need to regenerate them regularly unfortunately

**Reply by sergey-tihon** [reply]
Looks like I can generate token on behalf of fsprojects.
Are you sure that we want to provide centralised management for these tokens? Would be easier to add repo owner to package owners and allow then generate tokens on their's own
